nebula-components/tools/components/_flyout.scss

```diff
@@ -0,0 +1,33 @@
+@mixin nb-triangle($direction, $size: $nb-flyout-triangle-size, $color: $nb-flyout-triangle-color) {
```

Comment:
We should move this file into `tools/` and call it `_triangle.scss` as it's a global mixin.